### PR TITLE
Update to ODF 4.10 for RDR documentation

### PR DESCRIPTION
**Pull request by netzzer**

### CODE REVIEW

File: training/modules/ocs4/pages/odf4-multisite-ramen.adoc [hunk 1]
```diff
@@ -578,350 +583,187 @@ oc apply -f proxy-ca.yaml
...
+  ramen_manager_config.yaml: |
+    apiVersion: ramendr.openshift.io/v1alpha1
+    drClusterOperator:
+      deploymentAutomationEnabled: true  ## <-- Add to enable installation of ODR Cluster operator on managed clusters
+      catalogSourceName: redhat-operators
+      catalogSourceNamespaceName: openshift-marketplace
+      channelName: stable-4.10
+      clusterServiceVersionName: odr-cluster-operator.v4.10.0
+      namespaceName: openshift-dr-system
+      packageName: odr-cluster-operator
```
Comment (hatfieldbrian) [inline training/modules/ocs4/pages/odf4-multisite-ramen.adoc:674]: I think the configmap edit step may be removed from these instructions by setting some [`make [hub-]bundle` variables](https://github.com/red-hat-storage/ramen/blob/cb4cf7cf8b7544a53261f271df86a4c8665a1de5/Makefile#L256-L261) which result in these changes being applied to the configmap manifest included in the hub operator olm bundle.  I notice the catalog source name and namespace name are not variables in the `Makefile`, but I presume they could be added or even committed to the [downstream manifest](https://github.com/red-hat-storage/ramen/blob/cb4cf7cf8b7544a53261f271df86a4c8665a1de5/config/hub/manifests/ramen/ramen_manager_config_append.yaml#L7-L8) as they probably are unlikely to change.  However that could require merging with upstream @shyamsundarr

Comment (ShyamsundarR) [reply training/modules/ocs4/pages/odf4-multisite-ramen.adoc:676]: @hatfieldbrian line 655 states only to edit `deploymentAutomationEnabled: true ## <-- Add to enable installation of ODR Cluster operator on managed clusters` and the rest of the fields are already pre-set to ODF bundle values, does that address your comment?

Comment (hatfieldbrian) [reply training/modules/ocs4/pages/odf4-multisite-ramen.adoc:676]: Thanks @ShyamsundarR , I didn't realize the other fields were pre-set.  I presume `deploymentAutomationEnabled` is  pre-set to `false` or absent which is effectively false, and I wonder why it is not pre-set to `true` which remove the need for this edit step?

Comment (netzzer) [reply training/modules/ocs4/pages/odf4-multisite-ramen.adoc:676]: BZ to track https://bugzilla.redhat.com/show_bug.cgi?id=2059297

**Pull request merged**